node_modules/gh-orgunit/lib/internal/dao.js

```diff
@@ -982,8 +982,42 @@ var _updateOrCreateOrgUnit = function(appId, orgUnitToImport, parent, callback)
             'published': orgUnitToImport.published,
             'parentId': parent.id
         };
-        return createOrgUnit(appId, orgUnitToImport.displayName, orgUnitToImport.type, parent.GroupId, opts, callback);
+
+        // TODO: Refactor this check out of the DAO
+        _getOrCreateGroup(appId, orgUnitToImport.type, parent, function(err, groupId) {
```

Comment:
nm I get it. If it's a part, then we want it to have its own group created for it. If it's a module, then we want to use the parent course's group.